README.md

```diff
@@ -47,13 +47,11 @@ gantt
 ## 3rd party libraries
 
 The project uses the following 3rd party libraries:
-- `Open3d 0.18.0` for 3D point cloud processing it needs to be installed from source [from here](https://github.com/isl-org/Open3D/releases/download/v0.18.0/open3d-devel-windows-amd64-0.18.0.zip), unzip the file and by following the instructions below:
+- `Open3d 0.18.0` for 3D point cloud processing it needs to be installed from source [from here](https://github.com/isl-org/Open3D/releases/download/v0.18.0/open3d-devel-windows-amd64-0.18.0.zip), unzip the file to `Program Files/Open3D`. Change `<Your user name>` with your user name:
 ```terminal
-cd open3d
-mkdir build
-cmake -G "Visual Studio 16 2019" -A x64 -DCMAKE_INSTALL_PREFIX="C:\Program Files\Open3D" -S . -B build
-cmake --build build --config Release --target ALL_BUILD
-cmake --build build --config Release --target INSTALL
+Expand-Archive -DestinationPath 'C:\Users\<Your user name>\Downloads' -LiteralPath "C:\Users\<Your user name>\Downloads\open3d-devel-windows-amd64-0.18.0.zip"
+Move-Item -LiteralPath 'C:\Users\<Your user name>\Downloads\open3d-devel-windows-amd64-0.18.0\*' -Destination 'C:\Program Files\Open3D\'
+rmdir 'C:\Users\<Your user name>\Downloads\open3d-devel-windows-amd64-0.18.0\'
```

Comment:
With what you propose it will work with the current CMakeLists.txt file of DiffCheck because you probably already compiled everything before for Open3d (I remember you told me that you compiled). In this way we are using precompiled binaries and not compiled from source. It's safer for now to keep the compiled from source. But well spotted for the error in the link.

Reply:
Ok ! I probably made an error when compiling from source, then. For me unzipping the prebuilt binaries was the only way to make it work, that's why I made this PR. Thanks for the review !